src/docs/patterns/visualizations.mdx

```diff
@@ -1,62 +1,93 @@
 
 <DemoCard
     repository={'highcharts'}
     angular={`https://stackblitz.com/github/pxblue/highcharts/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/highcharts/tree/master/react-demo`}
 />
-
-PX Blue utilities for Highcharts graphs (line, column, pie, donut) are available as an NPM package ([@pxblue/highcharts](https://www.npmjs.com/package/@pxblue/highcharts)).
-This package includes helper functions to generate chart configurations using recommended PX Blue styles.
-These utilities also enable you to quickly create placeholder charts using simulated data. For more information about using Highcharts, check out their documentation.
-
-> Highcharts has different [licensing](https://shop.highsoft.com/developer-license) requirements depending on the nature of your application - you will need to contact Highsoft directly to ensure that your product is properly licensed. You can also see the [Highcharts Pricing](https://shop.highsoft.com/)) page for pricing information.
-
-### ChartJS
-
 <DemoCard
     repository={'chartjs'}
     angular={`https://stackblitz.com/github/pxblue/chartjs/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/chartjs/tree/master/react-demo`}
 />
 
-ChartJS is an open source alternative to HighCharts offering a scaled-down feature set that supports basic charting needs.
+## Common Pitfalls
+
+<ImageGrid images={[ClockwisePie]} regularWidth caption={'A pie chart should always be clockwise.'} />
 
-PX Blue utilities for ChartJS graphs (line, column, pie, donut) are available as an NPM package ([@pxblue/chartjs](https://www.npmjs.com/package/@pxblue/chartjs)). For more information, check out the [ChartJS Documentation](https://www.chartjs.org/).
+To avoid interfering with other UI elements, the charts should be built using primarily the [branding colors](/style/color#branding-colors).
 
-## Mapping (Mapbox)
+<React.Fragment>
+    <ImageGrid images={[DontUsePrimaryColor]} regularWidth caption={'Never use primary color palette in the charts.'} />
+    <Divider />
+</React.Fragment>
 
+# Maps
+
+<ImageGrid
+    regularWidth
+    images={[MapsImg]}
+    caption={[
+        'An example of a map, built with PX Blue dark theme.',
+        'The embedded map is a screenshot from OpenStreetMap, licensed under CC-BY-SA 2.0.',
+    ]}
+/>
 <DemoCard
     repository={'mapbox'}
     angular={`https://stackblitz.com/github/pxblue/mapbox/tree/master/angular-demo`}
     react={`https://codesandbox.io/s/github/pxblue/mapbox/tree/master/react-demo`}
 />
 
-If your application needs to show data on a geographical map, PX Blue supports the use of [Mapbox](https://www.mapbox.com/), a visually pleasing mapping solution that uses data from OpenStreetMap.
+Maps can be used to indicate geographical locations. They typically span across an entire screen.
+
```

Comment:
Is there anything we can add here or link out to to fill out this map section a little bit? It feels really light.

Reply:
Material Design does not have a proper page on the map topic.